Translations/Translations.csv

```diff
@@ -0,0 +1,88 @@
+"membercount_usage","[channel name]","[kanaal naam]"
+"membercount_brief","performs a member count.","Telt het aantal leden."
+"membercount_help","Perform a member count.\n\nWithout parameter: Returns the server member count.\n\nWith a channel name: Returns a count of all members currently in the given channel.\n\nWith parameter 'online': Return all members currently online.","Telt het aantal leden.\n\nZonder parameter: Telt het aantal leden in de server.\n\nMet een kanaal: Telt het aantal leden in dat kanaal.\n\nMet parameter 'online': Telt het aantal leden die momenteel online zijn."
+"membercount_online_result","There are {0} people online in this server!","Er zijn momenteel {0} gebruikers online in deze server!"
+"membercount_server_result","There are {0} people in {1}!","Er zitten momenteel {0} gebruikers in {1}!"
+"membercount_channel_nonexistant","The provided channel does not exist, foemp!","Het kanaal dat je meegaf bestaat niet, foemp!"
+"membercount_empty_channel","<#{0}> is empty, foemp!","<#{0}> is leeg, foemp!"
+"membercount_single_person","There is 1 person in channel <#{0}>!","Er is 1 persoon aanwezig in <#{0}>!"
+"membercount_channel_result","There are {0} people in channel <#{1}>!","Er zijn {0} mensen aanwezig in <#{1}>"
+"list_err_does_not_exit","This list does not exist, foemp!","Deze lijst bestaat niet, foemp!"
+"list_err_already_subscribed","<@{0}>, you are already subscribed to {1}, foemp!","<@{0}>, je ontvangt al notificaties voor {1}, foemp!"
+"list_err_not_subscribed","<@{0}>, you are not subscribed to {1}, foemp!","<@{0}>, je ontvangt geen notificaties voor {1}, foemp!"
+"list_subscribed","Subscribed <@{0}> to {1}.","<@{0}> ontvangt nu notificaties voor {1}."
+"list_unsubscribed","Unsubscribed <@{0}> from {1}.","<@{0}> ontvangt niet langer notificaties voor {1}."
+"list_err_empty","Nobody to notify, foemp!","Deze lijst is leeg, foemp!"
+"notifying","**{0}** Notified by <@{1}>\n","Ahem, dit is NerdlandBot uit de montagekamer, willen de volgende personen zich op vraag van <@{1}> melden voor **{0}** a.u.b.\n"
+"notify_message","With message: {0}","Met de volgende mededeling: {0}"
+"notify_sub_usage","[list name]","[lijst naam]"
+"notify_sub_brief","Subscribe to one or more list(s).","Abonneer op een of meerdere lijsten."
+"notify_sub_help","Subscribe to list(s)\n\nWithout a list name: you get an overview of all the lists with the corresponding emoji's. \nClick the emoji to subscribe to the corresponding list. \nClick the emoji again to unsubscribe from the corresponding list.\n\n With a list name: you will be subscribed to that list.","Abbonneer om notificaties te ontvangen\n\nZonder lijst: Je krijgt een overzicht van alle lijsten met de overeenkomstige emoji's.\nKies de emoji die overeenkomt met de gewenste lijst om je aan of af te melden\n\nMet lijst: Je zal notificaties ontvangen voor de gekozen lijst."
+"notify_unsub_usage","<list name>","<lijst naam>"
```

Comment:
I prefer not to, since that means a lot of edits if one of the usages change.
I personally do not mind duplicating keys, if they have a clearly different context.
(Ofcourse this would not be the case for yes, no, cancel, confirm, etc....)